assets/style.css

```diff
@@ -3,9 +3,23 @@ UI Styling
 */
 
 .background-image {
-  width: 100%;
-  height: 100%;
-  position: fixed;
-  z-index: -1;
-  opacity: 0.4;
-}
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: -1;
+    opacity: 0.4;
+  }
+
+  .input-btn {
+    margin-top: 15px !important;
+    height: 50px !important;
+  }
+
```

Comment:
!important should be used as a last resort, are we sure this is what is needed, same for all the importants

Reply:
agreed, i've never actually used important whats the goal of it?

Reply:
I accidentally left the top one in so I will take that one out, but unfortunately we need the other ones to overpower MD bootstrap. Here are pictures without the important on the height and the background color below:
<img width="1440" alt="Screen Shot 2020-08-20 at 11 27 22 AM" src="https://user-images.githubusercontent.com/63083817/90810739-511eae80-e2d8-11ea-9653-fcbfbeb44b53.png">
<img width="1440" alt="Screen Shot 2020-08-20 at 11 26 57 AM" src="https://user-images.githubusercontent.com/63083817/90810746-54199f00-e2d8-11ea-9eee-6e62d7a7e490.png">